webviz_subsurface/plugins/_multiple_regression.py

```diff
@@ -0,0 +1,1129 @@
+import warnings
+from itertools import combinations
+from pathlib import Path
+
+import dash
+import dash_core_components as dcc
+import dash_html_components as html
+import numpy as np
+import numpy.linalg as la
+import pandas as pd
+import plotly.graph_objects as go
+import statsmodels.api as sm
+import webviz_core_components as wcc
+from dash.dependencies import Input, Output, State
+from dash.exceptions import PreventUpdate
+from dash_table import DataTable
+from dash_table.Format import Format
+from webviz_config import WebvizPluginABC
+from webviz_config.common_cache import CACHE
+from webviz_config.utils import calculate_slider_step
+from webviz_config.webviz_store import webvizstore
+
+from .._datainput.fmu_input import load_parameters, load_csv, load_smry
+from .._utils.ensemble_handling import filter_and_sum_responses
+
+
+class MultipleRegression(WebvizPluginABC):
+    """ Visualizes the results of multiple regression of parameters and a chosen response using \
+forward selection to find the best fit.
+
```

Comment:
Not sure if we should claim "the best fit", or just something like "using forward selection to limit the number of terms". 

Think you can add some description additional description to the first part of the documentation ("the description tab"). E.g. that interaction terms are available, that the regression is linear, what is used as criterion for the forward selection, plus warnings/pitfalls (e.g. the case with correlated parameters).